utils/helpers.ts

```diff
@@ -184,3 +184,74 @@ export const cleanQuickstartName = (str: string): string =>
     .replace(/\s+/g, '-')
     .replace(/-+/, '-')
     .replace(/[^a-z0-9-]/g, '');
+
+/**
+ * Gets the unique base quickstart directory from a given file path.
+ * e.g. filePath: 'quickstarts/python/aiohttp/alerts/ApdexScore.yml' + targetChild: 'alerts' = 'python/aiohttp'.
+ * @param {String} filePath - Full file path of a file in a quickstart.
+ * @param {String} targetChild - Node in file path that should be preceded by a base quickstart directory.
+ * @return {String} Node in file path of the quickstart.
+ */
+const getQuickstartNode = (
+  filePath: string,
+  targetChild: string | undefined
```

Comment:
Potential for optional parameter syntax (if you want):

```suggestion
  targetChild?: string
```

Reply:
Done!